contracts/LifToken.sol

```diff
@@ -3,6 +3,13 @@ pragma solidity ^0.4.13;
 import "zeppelin-solidity/contracts/token/MintableToken.sol";
 import "zeppelin-solidity/contracts/lifecycle/Pausable.sol";
 
+/**
+ * @title Líf, the Winding Tree token
+ *
+ * Implementation of Líf, the ERC20 token for Winding Tree, with extra methods
+ * for transfers with contract calls.
```

Comment:
what about "to transfer value and data allowing the execution of calls in transfer" ? or smth like that

Reply:
Even though I like "contract call" or "function call" more, I'm fine with "execution of calls". However, it should be singular like in "execution of a call on transfer", right? It's a single call, not multiple calls

Reply:
yes, singular.